src/builder.js

```diff
@@ -516,6 +517,22 @@ export default function builder() {
     instrument(window, 'setTimeout', instrumentTimer);
     instrument(window, 'setInterval', instrumentTimer);
 
+    // Breadcrumbs: instrument click events
+    window.addEventListener('click', (event) => {
+      let message
+      try {
+        message = stringNameOfElement(event.target)
+      } catch(e) {
+        message = '[unknown]'
+      }
+
+      if (message.length === 0) return
+
+      self.addBreadcrumb(message, {
+        category: 'ui.click'
```

Comment:
@rabidpraxis is there a specific naming convention you'd like to use for subcategories, or is this OK? We might add additional UI-related events in the future, like key presses, input, focus, etc.

Reply:
Yeah, this makes sense to me. So far I have kept the categories pretty simple, but I like the idea of subcategories. The only current usage of the category is to choose an icon in the interface.